Tasks/AzureCLIV2/task.json

```diff
@@ -0,0 +1,198 @@
+    "category": "Deploy",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "demands": [],
+    "version": {
+        "Major": 2,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "minimumAgentVersion": "2.0.0",
+    "instanceNameFormat": "Azure CLI $(scriptPath)",
+    "showEnvironmentVariables": true,
+    "groups": [
+        {
+            "name": "advanced",
+            "displayName": "Advanced",
+            "isExpanded": true
+        }
+    ],
+    "inputs": [
+        {
+            "name": "connectedServiceNameARM",
+            "aliases": [
+                "azureSubscription"
+            ],
+            "type": "connectedService:AzureRM",
+            "label": "Azure temp subscription",
+            "required": true,
+            "helpMarkDown": "Select an Azure resource manager subscription for the deployment",
+            "properties": {
+                "EndpointFilterRule": "ScopeLevel != ManagementGroup"
+            }
+        },
+        {
+            "name": "scriptType",
+            "type": "pickList",
+            "label": "Script Type",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Type of script: Bat/Shell/PowerShell/PowerShell Core script. Select Shell/PowerShell Core script when running on Linux agent or Batch/PowerShell/PowerShell Core script when running on Windows agent. PowerShell Core script can run on cross-platform agents (Linux, macOS, or Windows).",
+            "options": {
+                "batch": "Batch script",
+                "bash": "Shell script",
+                "ps": "Powershell script",
+                "pscore": "Powershell Core script"
+            }
+        },
+        {
+            "name": "scriptLocation",
+            "type": "pickList",
+            "label": "Script Location",
+            "defaultValue": "scriptPath",
+            "required": true,
+            "helpMarkDown": "Path to script: File path or Inline script",
+            "options": {
+                "inlineScript": "Inline script",
+                "scriptPath": "Script path"
+            }
+        },
+        {
+            "name": "scriptPath",
+            "type": "filePath",
+            "label": "Script Path",
+            "defaultValue": "",
+            "required": true,
+            "visibleRule": "scriptLocation = scriptPath",
+            "helpMarkDown": "Fully qualified path of the script(.bat or .cmd or .ps1 when using Windows based agent else .sh or .ps1 when using linux based agent) or a path relative to the the default working directory"
```

Comment:
I think we need not specify file extensions here. Anyways user has chosen a script type earlier. We should just say choose file as per script type specified above